code/modules/martial_arts/cqc.dm

```diff
@@ -4,6 +4,7 @@
 	combos = list(/datum/martial_combo/cqc/slam, /datum/martial_combo/cqc/kick, /datum/martial_combo/cqc/restrain, /datum/martial_combo/cqc/pressure, /datum/martial_combo/cqc/consecutive)
 	var/restraining = FALSE //used in cqc's disarm_act to check if the disarmed is being restrained and so whether they should be put in a chokehold or not
 	var/chokehold_active = FALSE //Then uses this to determine if the restrain actually goes anywhere
+	var/datum/action/defensive_stance/defensive = new/datum/action/defensive_stance()
```

Comment:
Please create this reference on teach, this will currently GC fail otherwise
Please see https://github.com/ParadiseSS13/Paradise/pull/19212 as an example of how to do this